pset-iterators.js

```diff
@@ -92,11 +99,13 @@ Tip: your new array should contain 45 elements
 console.log('Problem 3:')
 
 // Add your code below this line
-
+const presidentialParties = presidents.map(function(president) {
+	return president.party
+});
 // Add your code above this line
 
 /** added for formatting purposes **/
-console.log('')
+console.log(presidentialParties)
 console.log('-----------------')
```

Comment:
@jl1982 Great work using the map method to return the president's party. 

Take a look at this approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L94